back/app/services/initiative_status_service.rb

```diff
@@ -78,9 +78,8 @@ class InitiativeStatusService
     }
   }
 
-  def transition_allowed?(_initiative, status1, status2, with_feedback: false)
-    transition_possibility = MANUAL_TRANSITIONS.dig status1.code, status2.code
-    transition_possibility && (!transition_possibility[:feedback_required] || with_feedback)
```

Comment:
Why are we removing the `:feedback_required` check? I guess there is an intention...

Is it because we don't require feedback on proposal idea manual status changes now?
Or maybe we always require feedback and this functionality lives elsewhere now?

Reply:
It's been a while ago, but we basically decided to change the behaviour and simplify it. We are no longer requiring official feedback to be posted for any status transition anymore. The UI will still ask the moderators if they want to also give feedback when they change the status, but it will always be possible to say no and transition without feedback. As we no longer have the `changes_required` status, there is no more status that makes no sense without an official feedback.